js/models/messages.js

```diff
@@ -655,6 +655,8 @@
           !this.get('isPublic') ||
           this.getConversation().isModerator(this.OUR_NUMBER) ||
           this.getSource() === this.OUR_NUMBER,
+        hasModPerms:
```

Comment:
This seems very similar to `isModerator`

Reply:
isModerator checks if message sender isModerator,
hasModPerms checks mod status of person performing the ban on the message

Reply:
Good catch, Vince.
@msgmaxim  should we rename `isModerator` to `senderIsModerator` to clear up any confusion?

Reply:
True. Then yeah, we need to rename these to something better to avoid the confusion.